docs/book/v5/transports.md

```diff
@@ -0,0 +1,26 @@
+# Transports
+
+`dot-mail` can use any transport class that implements `symfony\Component\Mailer\Transport\TransportInterface`, with the standard transport available being:
+
+- `Laminas\Mail\Transport\Smtp`
+
+- Note: feel free to use any custom transport you desire, provided it implements the mentioned `TransportInterface`.
```

Comment:
```suggestion
> Feel free to use any custom transport you desire, provided it implements the mentioned `TransportInterface`.
```

Reply:
Please clarify that only SENDMAIL and SMTP are used.